esmvaltool/diag_scripts/mpqb/mpqb_lineplot.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+"""Python example diagnostic."""
+import logging
+import os
+from pprint import pformat
+
+import iris
+import matplotlib.dates as mdates
+import matplotlib.pyplot as plt
+from cf_units import Unit
+from mpqb_utils import get_mpqb_cfg
+
+from esmvaltool.diag_scripts.shared import group_metadata, run_diagnostic
+from esmvaltool.diag_scripts.shared._base import (
+    ProvenanceLogger,
+    get_plot_filename,
+)
+
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def get_provenance_record(caption):
+    """Create a provenance record describing the diagnostic data and plot."""
+    record = {
+        'caption': caption,
+        'statistics': ['mean'],
+        'domains': ['user'],
+        'plot_type': 'lineplot',
+        'authors': [
+            'mueller_benjamin',
+            'crezee_bas',
+            'hassler_birgit',
+        ],
+        'projects': ['cmug'],
+        'references': [
+            'acknow_project',
+        ],
+    }
+    return record
+
+
+def _unify_time_coord(cube):
+    """Unify time coordinate of cube."""
+    if not cube.coords('time', dim_coords=True):
+        return
+    time_coord = cube.coord('time')
+    dates_points = time_coord.units.num2date(time_coord.points)
+    dates_bounds = time_coord.units.num2date(time_coord.bounds)
+    new_units = Unit('days since 1850-01-01 00:00:00')
+    new_time_coord = iris.coords.DimCoord(
+        new_units.date2num(dates_points),
+        bounds=new_units.date2num(dates_bounds),
+        var_name='time',
+        standard_name='time',
+        long_name='time',
+        units=new_units,
+    )
+    coord_dims = cube.coord_dims('time')
+    cube.remove_coord('time')
+    cube.add_dim_coord(new_time_coord, coord_dims)
+
+
+def main(cfg):
+    """Create lineplot."""
+    ylims = [cfg.pop('y0', None), cfg.pop('y1', None)]
```

Comment:
It looks like `y0` and `y1` are possible optional options for this diagnostic which are not listed in the documentation